bcgov/HOUS-permit-portal
# [bphh-917] Move email and phone requirement field setup to above contact requirements

[bphh-917] Move email and phone requirement field setup to above contact requirements

## Description



## What type of PR is this? (check all applicable)

- [X ] 🐛 Bug Fix

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-917


## Steps to QA
- log in as super admin
- Go to requirements library
- Select any block to edit
- Click add
- The email and phone types should be right before contact types
